bson/bsoncodec/struct_tag_parser_test.go

```diff
@@ -13,12 +13,13 @@ import (
 	"github.com/google/go-cmp/cmp"
 )
 
-func TestDefaultStructTagParser(t *testing.T) {
-	testCases := []struct {
+func TestStructTagParsers(t *testing.T) {
+	type testCase struct {
```

Comment:
nit: Can we change this back to `testCases := []struct{}{...}` to keep consistency with other tests?

Reply:
done